src/api/auth.ts

```diff
@@ -118,6 +118,12 @@ export function getTeamUsers(team: string, organization: string, auth: any): any
   return users;
 }
 
+export function checkPackageName(pkgName: string): boolean {
+  let regexStart = /^[a-z0-9~()'!*:@,;+?-]{1}/g;
```

Comment:
Why do you think that would be better? I'll remove `~:?` from each regex and check the whole word. If there is a match that means that this characters doesn't exists there. I'll add example like that in unit test...